pkg/cloud/cloud.go

```diff
@@ -535,7 +542,20 @@ func newBatcherManager(svc util.EC2API) *batcherManager {
 }
 
 // execBatchDescribeVolumes executes a batched DescribeVolumes API call depending on the type of batcher.
-func execBatchDescribeVolumes(svc util.EC2API, input []string, batcher volumeBatcherType) (map[string]*types.Volume, error) {
+func execBatchDescribeVolumes(svc util.EC2API, input []string, batcher volumeBatcherType, c *cloud) (map[string]*types.Volume, error) {
```

Comment:
Don't pass in all of cloud, pass in the map you need, same as we do with the EC2API param
(obviously ditto all the other functions this was changed in)